### PR TITLE
Feed cleanup

### PR DESCRIPTION
**Pull request by mmitche**
Switch away from blob feeds and general clenaup

### CODE REVIEW

File: NuGet.config [hunk 1]
```diff
@@ -2,9 +2,8 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="arcade" value="https://dotnetfeed.blob.core.windows.net/dotnet-tools-internal/index.json" />
-    <add key="roslyn-tools" value="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
```
Comment (jmarolf) [inline NuGet.config]: we need to keep `roslyn-tools` for this repo

Comment (mmitche) [reply NuGet.config]: Is it taking regular updates from that repo? If it's one-off stuff I can probably push those packages. We may lose myget sometime in the future.

Comment (jmarolf) [reply NuGet.config]: this is the package: https://dotnet.myget.org/feed/roslyn-tools/package/nuget/UpdateTemplateVersion

I do not update it regularly. We can move it whereever

Comment (mmitche) [reply NuGet.config]: You're uploading manually? If so, then I think it's pretty trivial to just upload to dotnet-eng instead (you should have permissions) and thus we can deprecate this feed.

Comment (jmarolf) [reply NuGet.config]: never done an upload to dnceng before. myget was two clicks all told. Where are the docs for dnceng packages?

Comment (mmitche) [reply NuGet.config]: https://dev.azure.com/dnceng/public/_packaging?_a=connect&feed=dotnet-eng

Once you install the credential provider for AzDO, it's even simpler, and it auths for you.

```
nuget push --source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json <package>
```

Comment (mmitche) [reply NuGet.config]: Sorry:

```
dotnet nuget push --source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json --api-key az <package-path>
```

Comment (jmarolf) [reply NuGet.config]:
```cmd
λ dotnet nuget push --source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json --api-key az UpdateTemplateVersion.1.0.4-beta1.nupkg
Pushing UpdateTemplateVersion.1.0.4-beta1.nupkg to 'https://pkgs.dev.azure.com/dnceng/9ee6d478-d288-47f7-aacc-f6e6d082ae6d/_packaging/1a5f89f6-d8da-4080-b15f-242650c914a8/nuget/v2/'...
  PUT https://pkgs.dev.azure.com/dnceng/9ee6d478-d288-47f7-aacc-f6e6d082ae6d/_packaging/1a5f89f6-d8da-4080-b15f-242650c914a8/nuget/v2/
  Unauthorized https://pkgs.dev.azure.com/dnceng/9ee6d478-d288-47f7-aacc-f6e6d082ae6d/_packaging/1a5f89f6-d8da-4080-b15f-242650c914a8/nuget/v2/ 456ms
error: Response status code does not indicate success: 401 (Unauthorized).
```

Says I am not authorized, I followed the directions to install the credential provider but it does not change the output.

**Comment by mmitche** [comment]
> Blocking this PR until we can update the nuget package source. I apparently do not have permissions.



> Blocking this PR until we can update the nuget package source. I apparently do not have permissions.

Try now, I added you as a contributor to that feed.

**Comment by jmarolf** [comment]
@mmitche thanks! I will re-run the build and merge when green

**Pull request merged**